src/lib/components/Search/index.js

```diff
@@ -341,7 +344,13 @@ class Search extends BaseElement {
     }
     try {
       const index = await loadAlgoliaLibrary();
-      const {hits} = await index.search(query, {hitsPerPage: 10});
+      const {hits} = await index.search(query, {
+        hitsPerPage: 10,
+        attributesToHighlight: ['title'],
+        attributesToRetrieve: ['url'],
+        highlightPreTag: '<mark>',
+        highlightPostTag: '</mark>',
```

Comment:
```suggestion
        highlightPreTag: '<strong>',
        highlightPostTag: '</strong>',
```

going with strong here, it's consistent with google search (which uses `b`)